admin_guide/install/advanced_install.adoc

```diff
@@ -97,16 +75,56 @@ Removing this line til we have a better instruction:
 # git checkout -b 3.x v3.0.0
 ////
 
+[[configuring-ansible]]
+
 == Configuring Ansible
 
-You must create an *_/etc/ansible/hosts_* file for the playbook to use during
-the installation. The following is an example of a Bring Your Own (BYO) host
-inventory, based on the link:#default-hosts[example host environment]. You can
-see these example hosts present in both the *[masters]* and *[nodes]* sections:
+You must create an *_/etc/ansible/hosts_* file, which is Ansible's inventory
+file for the playbook to use during the installation.
+
+Any hosts you designate as masters during the installation process should also
+be configured as nodes that are marked
+link:../../admin_guide/manage_nodes.html#marking-nodes-as-unschedulable-or-schedulable[unschedulable].
```

Comment:
@adellape Would it be better to say:

"Any hosts you designate as masters during the installation process should also be configured as unschedulable nodes."

I feel this may be impacted by my understanding of the 'unschedulable' term, tho...